.prettierignore

```diff
@@ -24,6 +24,9 @@ contracts/src/v0.5
 contracts/src/v0.6
 contracts/**/vendor
 
+# Ignore generated GQL files
+operator_ui/graphql.schema.json
```

Comment:
This doesn't look to be working as we have a committed file?

Reply:
This is for `prettier` as the generated file does not pass the prettier CI check

Reply:
This has been removed now that we do not commit the generated files